x-pack/test/apm_api_integration/tests/metrics_charts/serverless.spec.ts

```diff
@@ -0,0 +1,244 @@
+        });
+
+        it('returns correct overallValue on System memory usage chart', () => {
+          const metric = metrics.charts.find((chart) => chart.key === 'memory_usage_chart');
+          expect(metric).not.to.be.empty();
+
+          const memoryValue = roundNumber(1 - MEMORY_FREE / MEMORY_TOTAL);
+
+          const maxMemorySeries = metric?.series.find(({ title }) => title === 'Max');
+
+          expect(maxMemorySeries).not.to.be.empty();
+          expect(roundNumber(maxMemorySeries?.overallValue)).to.equal(memoryValue);
+
+          const avgMemorySeries = metric?.series.find(({ title }) => title === 'Average');
+          expect(avgMemorySeries).not.to.be.empty();
+          expect(roundNumber(avgMemorySeries?.overallValue)).to.equal(memoryValue);
+        });
+
+        it('returns correct overallValue on Compute usage chart', () => {
+          const metric = metrics.charts.find((chart) => chart.key === 'compute_usage');
+          expect(metric).not.to.be.empty();
+          const bytesMs = MEMORY_TOTAL * BILLED_DURATION_MS;
+          const gbSecs = bytesMs / (1024 * 1024 * 1024 * 1000);
+          expect(metric?.series[0].overallValue).to.equal(gbSecs);
+        });
+
+        it('returns correct overallValue on Active instances chart', () => {
+          const metric = metrics.charts.find((chart) => chart.key === 'active_instances');
+          expect(metric).not.to.be.empty();
+          expect(metric?.series[0].overallValue).to.equal(1);
+        });
+
+        it('returns correct overallValue on cold start count chart', () => {
+          const metric = metrics.charts.find((chart) => chart.key === 'cold_start_count');
+          expect(metric).not.to.be.empty();
+          expect(metric?.series[0].overallValue).to.equal(30);
+        });
+      });
+
+      describe('nodejs', () => {
+        let metrics: APIReturnType<'GET /internal/apm/services/{serviceName}/metrics/charts'>;
+        before(async () => {
+          const { status, body } = await callApi('lambda-node', 'nodejs');
+          expect(status).to.be(200);
+          metrics = body;
+        });
+
+        it('returns all metrics chart', () => {
+          expect(metrics.charts.length).to.be.greaterThan(0);
+          expect(metrics.charts.map(({ title }) => title).sort()).to.eql([
+            'Active instances',
+            'Avg. Duration',
+            'Cold start',
+            'Cold start duration',
+            'Compute usage',
+            'System memory usage',
+          ]);
+        });
+
+        it('returns correct overallValue on Cold start chart', () => {
+          const metric = metrics.charts.find((chart) => chart.key === 'cold_start_duration');
+          expect(metric).not.to.be.empty();
+          expect(metric?.series[0].overallValue).to.equal(COLD_START_DURATION_NODE);
+        });
+
+        it('returns correct overallValue on cold start count chart', () => {
```

Comment:
what is this testing? I don't see a value assertion?

Reply:
fixed it here https://github.com/elastic/kibana/pull/139041/commits/cfe36ac632b3fa1a6e1b54868fafa981e14df8dd#diff-c4258177e36431d334f358c863471cc966bc383bf0816383ca2abddee3810123R423